roxie/ccd/ccdmain.cpp

```diff
@@ -1243,8 +1243,14 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         initializeTopology(topoValues, myRoles);
 #endif
         createDelayedReleaser();
-        globalPackageSetManager = createRoxiePackageSetManager(standAloneDll.getClear());
-        globalPackageSetManager->load();
+        Owned<IRoxieDaliHelper> daliHelper;
+        if (oneShotRoxie)
+            daliHelper.setown(connectToDali(ROXIE_DALI_CONNECT_TIMEOUT));
```

Comment:
@richardkchapman  There are a couple of places that assume globalPackageSetManager is initialised.  e.g., CQueryFactory::lookupLibrary and a few others.  Will this change cause problems there?